### ISSUE TITLE
Increase sensitivity to stop sooner the longer it teases.

### ISSUE THREAD

**Issue by LanceLake** [MausTec/nogasm-wifi#50]
It would be cool to have a setting in the JSON file that is an decimal (default of 0.00) that will lower "sensitivity_threshold" by that amount for every edge. This will allow for the device to use the stated sensitivity in the beginning, but as time goes on, the machine gets more and more sensitive to the users arousal and stops sooner and sooner. So the effect of being edged multiple times making it more likely to orgasm is minimized. 

You may also have to set in the JSON a bottom limit for this so that it doesn't reach 0 and essentially turn off. Default it to 10 or so.

**Comment by MauAbata** [comment]
I agree, as well as the request to add in cooldown periods and such. I commented over on #52 about how I'm adding in an event-based plugin system so that requests like this can be loaded up as community code / json files to change how the device works, vs. being baked into the firmware and pending a whole release cycle. 

Once that system is in place, I'll write up some example plugins that cover this feature and others. Please look forward to it™️!
